.github/workflows/elvis_lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: Elvis Linter
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          echo "$GITHUB_BASE_REF"
+          echo "$GITHUB_HEAD_REF"
+          set +e
+          curl -LO https://github.com/emqx/elvis/releases/download/1.0.0-emqx-1/elvis
```

Comment:
in the script, we should set elvis version in a variable ELVIS_VERSION='1.0.0-emqx-1'
then download elvis only when it's missing or version mismatch.

```
if [ ! -f ./elvis ] || [ "$(./elvis -v | grep -oE '[1-9]+\.[0-9]+\.[0-9]+\-emqx-[0-9]+')" != "$ELVIS_VERSION" ]; then 
    curl ....
fi
```